lib/stripe_mock/request_handlers/helpers/subscription_helpers.rb

```diff
@@ -52,6 +52,24 @@ def custom_subscription_params(plans, cus, options = {})
           params.merge!({status: 'trialing', current_period_end: end_time, trial_start: start_time, trial_end: end_time, billing_cycle_anchor: nil})
         end
 
+        # Support updating subscription with multiple plan items
```

Comment:
Just noting for posterity: change from https://github.com/docsend/stripe-ruby-mock/pull/1/files#diff-ce78c328fa7b9205861fad8c25d908164c4e6f7a5d1fbd2caecef970418046afR27